app/models/github/issue_label.rb

```diff
@@ -3,4 +3,37 @@ class Github::IssueLabel < ApplicationRecord
     inverse_of: :labels,
     foreign_key: "github_issue_id",
     class_name: "Github::Issue"
+
+  def self.type_prefix(type)
+    "x:#{type}"
```

Comment:
```suggestion
    "x:#{type}/"
```

Reply:
I think it's marginally safer to include the `/` here, in case (stupidly) one label's name becomes a subset of the other. It's probably a meaningless change in reality, but just gives a tiny bit more safety.